sdk/modelsrepository/Azure.Iot.ModelsRepository/src/DependencyResolutionOption.cs

```diff
@@ -4,23 +4,24 @@
 namespace Azure.Iot.ModelsRepository
 {
     /// <summary>
-    /// The dependency processing options.
+    /// The model dependency resolution options.
     /// </summary>
     public enum DependencyResolutionOption
     {
         /// <summary>
-        /// Do not process external dependencies.
+        /// Try to get pre-computed model dependencies using .expanded.json.
+        /// If the model expanded form does not exist fall back to normal dependency processing.
```

Comment:
What is normal processing that we fall back to?

Reply:
Updated the wording a bit for this enum which tries to explain this better in the recently opened PR https://github.com/Azure/azure-sdk-for-net/pull/19072